lit/setup-and-operations/install.lit

```diff
@@ -68,6 +68,232 @@ page.
   different machine than the server.
 }
 
+
+\section{
+  \title{Configuring Auth Providers}{auth-config}
+
+  At startup, you will need to configure any local users or third-party auth
+  providers that you wish to use with Concourse.
+
+  In contrast to previous releases of Concourse, \code{users} are now central
+  to the authentication flow. Instead of logging in as a team, you now log in
+  as a user and can belong to one or more teams. Users can be added to a team
+  by configuring the team's whitelist as described in
+  \reference{authentication}.
+
+  \section{
+    \title{Local Auth}{local-auth-config}
+
+    In order to add new users to the system, you can supply the
+    \code{--add-local-user} flag to the \code{concourse web} subcommand like
+    so:
+
+    \codeblock{bash}{{{
+    concourse web \
+      --add-local-user myuser:mypass
+    }}}
+  }
+
+  \section{
+    \title{GitHub Auth}{github-auth-config}
+
+    A Concourse server can authenticate against GitHub to take advantage of
+    their permission model and other security improvements in their
+    infrastructure.
+
+    \section{
+      \title{Creating a GitHub application}
+
+      First you need to \link{create an OAuth application on
+      GitHub}{https://github.com/settings/applications/new}.
+
+      The callback URL will be the external URL of your Concourse server with
+      \code{/sky/issuer/callback} appended. For example, Concourse's own CI
+      server's callback URL would be
+      \code{https://ci.concourse-ci.org/sky/issuer/callback}.
+    }
+
+    \section{
+      \title{Configuring the client}
+
+      You will be given a Client ID and a Client Secret for your new
+      application. These will then be passed to the following flags:
+
+      \list{
+        \code{--github-client-id=CLIENT_ID}
+      }{
+        \code{--github-client-secret=CLIENT_SECRET}
+      }
+
+      If you're configuring GitHub Enterprise, you'll also need to set the
+      following flags:
+
+      \list{
+        \code{--github-host=github.example.com}
+      }{
+        \code{--github-ca-cert=/tmp/some-cert}
+      }
+
+      The GitHub Enterprise host should not contain a scheme, or a trailing
+      slash.
+
+      \codeblock{bash}{{{
```

Comment:
This codeblock is kind of without context. It can probably just be omitted if we change the above to specify that the flags go to `web`.